### PR TITLE
neutron: fix infoblox migration (bsc#1020121)

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by rsalevsky** [comment]
Would be good to cherry-pick this change from https://github.com/crowbar/crowbar-openstack/pull/765.

**Comment by dirkmueller** [comment]
@rsalevsky its a different migration (different number) so I opted for not cherry-pick x'ing

File: chef/data_bags/crowbar/migrate/neutron/049_infoblox_update.rb [hunk 1]
```diff
@@ -9,7 +9,7 @@ def upgrade(ta, td, a, d)
   # won't touch it. If, on the other hand, we are dealing with the Cloud 5
   # version from October 2015 we'll just overwrite it since the infoblox plugin
   # changed a great deal since then.
-  unless (a.key? "infoblox") && (a[:infoblox].key? "grid_defaults")
+  unless (a.key?("infoblox") && a[:infoblox].key?("grid_defaults")
```
Comment (rsalevsky) [inline chef/data_bags/crowbar/migrate/neutron/049_infoblox_update.rb]: The first `(` is now too much.

Comment (dirkmueller) [inline chef/data_bags/crowbar/migrate/neutron/049_infoblox_update.rb]: Wow, so many ways to get it wrong :( thanks!

**Pull request merged**